backports/py-protobuf/APKBUILD

```diff
@@ -0,0 +1,55 @@
+# Maintainer: Corentin Henry <corentinhenry@gmail.com>
+# Contributor: Corentin Henry <corentinhenry@gmail.com>
+pkgname=py-protobuf
+_pkgname=${pkgname#py-}
+pkgver=3.5.2
+pkgrel=1
+pkgdesc="Google's data interchange format."
+url="https://github.com/google/protobuf"
+arch="noarch"
+license="BSD-3-Clause"
+depends="py-six>=1.9"
+makedepends="python2-dev python3-dev py-setuptools py-google-apputils"
```

Comment:
It's upstream problem, but google-apputils (supports only py2) was used in protobuf 2.6 era; not needed any more.